src/main/java/cloud/orbit/actors/cluster/impl/RedisOrbitClient.java

```diff
@@ -30,30 +30,32 @@
 
 import org.redisson.api.RedissonClient;
 import org.redisson.api.listener.MessageListener;
-import org.redisson.connection.ConnectionListener;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
 
 import cloud.orbit.tuples.Pair;
 
-import java.net.InetSocketAddress;
 import java.util.ArrayList;
 import java.util.List;
 import java.util.Timer;
 import java.util.TimerTask;
+import java.util.stream.Collectors;
 
 public class RedisOrbitClient
 {
+    private static Logger logger = LoggerFactory.getLogger(RedisOrbitClient.class);
+
     private final RedissonClient redisClient;
-    private final Integer messagingHealthcheckInterval;
-    private boolean isConnected = false;
+    private volatile boolean isConnected = false;
     private final List<Pair<String, MessageListener<Object>>> subscriptions = new ArrayList<>();
-    private Timer connectionTimer;
+    private final Timer connectionTimer;
 
     public RedisOrbitClient(final RedissonClient redisClient, final Integer messagingHealthcheckInterval)
     {
-        this.messagingHealthcheckInterval = messagingHealthcheckInterval;
         this.redisClient = redisClient;
         this.isConnected = redisClient.getNodesGroup().pingAll();
         this.connectionTimer = new Timer();
+
```

Comment:
nit: remove lines that weren't changed (when you were putting the lock in). Shows up a few times in the PR and is noise on the actual changes.

Reply:
Should be cleaned up now.